daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md

```diff
@@ -158,8 +171,13 @@ The latest Dapr helm chart no longer supports Helm v2. Please migrate from Helm
     --wait
     ```
 
+1. Install the Dapr Dashboard:
```

Comment:
This is not really step 4. It is its own sub heading where you can choose this new chart that still includes the dapr dashboard instead of the dapr/dapr chart. So you can say something like this

## Installing the Dapr dashboard as part of the control plane
If you want to install the Dapr dashboard use this Helm chart with the additional settings of your choice.

`helm install dapr dapr/dapr-dashboard --namespace dapr-system`

********
And for full context it would look like this
```
helm repo add dapr https://dapr.github.io/helm-charts/
helm repo update
kubectl create namespace dapr-system
helm install dapr dapr/dapr-dashboard --namespace dapr-system
```